### DISCUSSION TITLE
Android: content not fit cutout area

### DISCUSSION THREAD

**Discussion by lyaaz** [rustdesk/rustdesk#9228] [General]
### Bug Description

When using an Android device to remotely control another device (especially with the same resolution), content cannot be displayed in the cutout area, which instead appears as a black border.

### How to Reproduce

1. Use an Android device with a cutout area to remotely control another device (especially when both devices have the same resolution).
2. Notice that the screen shows black borders and the content is scaled down.

### Expected Behavior

The content should be displayed in the cutout area.

### Operating system(s) on local side and remote side

Android 14 -> Android/Windows 10

### RustDesk Version(s) on local side and remote side

1.3.0 -> 1.3.0

### Screenshots

The black bar appears on both landscape and portrait modes.
![markup_1000015893](https://github.com/user-attachments/assets/bcf50134-72fb-4629-a8b4-27e9bda601f2)

**Comment by rustdesk** [comment]
I see anydesk has the same behavior.

**Reply by lyaaz** [reply]
But when the two devices have the same resolution (or aspect ratio), this behavior will result in the inability to display in full screen, and the content will always be scaled. Maybe we can add an option to control this behavior
